src/main/java/org/jenkinsci/plugins/docker/workflow/FromFingerprintStep.java

```diff
@@ -99,7 +99,7 @@ public String getToolName() {
                         }
                         if (line.startsWith("FROM ")) {
                             fromImage = line.substring(5);
-                            break;
+                            continue;
```

Comment:
Arguably would be better to keep a `List<String> fromImages` and fingerprint them all, just skipping the ` as …` part of the line. (Even better would be to create distinct kinds of facets representing multistage builds.) Anyway there is no maintained code consuming these fingerprints so it matters little.